src/js/components/Chart/Chart.js

```diff
@@ -415,7 +445,16 @@ const Chart = React.forwardRef(
           };
 
           return (
-            <g key={key} stroke="none">
+            <g
+              key={key}
+              stroke="none"
+              fill={valueColor ? normalizeColor(valueColor, theme) : undefined}
+              opacity={
+                valueOpacity
+                  ? theme.global.opacity[valueOpacity] || valueOpacity
+                  : undefined
+              }
```

Comment:
```suggestion
              opacity={
                valueOpacity && theme.global.opacity[valueOpacity] || valueOpacity
              }
```
Would that work cleaner?

Reply:
yes, done